### ISSUE TITLE
TypeScript types for callback arguments don't match implementation

### ISSUE THREAD

**Issue by OzzieOrca** [AppsFlyerSDK/appsflyer-react-native-plugin#147]
# Report
For both `setAdditionalData` and `setCustomerUserId` (and maybe others, these are the only 2 functions I've tested), the types indicate the second parameters (the `successC` callback) are optional.
https://github.com/AppsFlyerSDK/appsflyer-react-native-plugin/blob/master/index.d.ts#L56
https://github.com/AppsFlyerSDK/appsflyer-react-native-plugin/blob/master/index.d.ts#L58

But I get runtime errors unless I pass a real function.
<img width="504" alt="Screen Shot 2020-04-06 at 10 50 59 AM" src="https://user-images.githubusercontent.com/756501/78594324-f12ddc80-77fc-11ea-9637-ed3a255d413c.png">
Making these changes seems to fix the error.
```diff
- appsFlyer.setCustomerUserId(userId);
+ appsFlyer.setCustomerUserId(userId, () => {});
- appsFlyer.setAdditionalData({ someId: someId });
+ appsFlyer.setAdditionalData({ someId: someId }, () => {});
```

Ideally the library should be able to handle null callback functions. But if these are truly required, the types should be updated to reflect that.

## Plugin Version
5.1.3

## On what Platform are you having the issue?
iOS and Android

## What did you do?
Called `setCustomerUserId` and `setAdditionalData` with one argument.

## What did you expect to happen?
No errors to be thrown

## What happened instead?
Raised a native error over the bridge about being unable to invoke the callback functions

## Please provide any other relevant information.
Happened on initial install of library. Didn't notice one of the errors raised until QA.

**Comment by af-fess** [comment]
@OzzieOrca thank you, we will check it

**Comment by af-vs** [comment]
This was fixed for Android, doesn't seem possible to make callbacks optional on iOS though...

**Comment by OzzieOrca** [comment]
Thanks :) I'm not sure how it'd be not possible on iOS. On the JS side in this library you could make the default value be an empty function so the library consumer doesn't have to specify one. Or you could change the types to require it...

**Comment by amit-kremer93** [comment]
Hey @OzzieOrca!
the issue will be fixed in the next version of the plugin (v6.0.31)
